### ISSUE TITLE
It Doesn't Work when Sending Email with Attachment

### ISSUE THREAD

**Issue by triasrahman** [mplodowski/maillog-plugin-public#3]
Hi. I got an error when sending email with attachment. Here is the stack trace.

```
Error: Call to undefined method Symfony\Component\Mime\Part\DataPart::getFilename() in /var/www/html/plugins/renatio/maillog/listeners/LogEmail.php:50
Stack trace:
#0 [internal function]: Renatio\MailLog\Listeners\LogEmail->Renatio\MailLog\Listeners\{closure}(Object(Symfony\Component\Mime\Part\DataPart), 0)
#1 /var/www/html/vendor/laravel/framework/src/Illuminate/Collections/Arr.php(560): array_map(Object(Closure), Array, Array)
#2 /var/www/html/vendor/laravel/framework/src/Illuminate/Collections/Collection.php(768): Illuminate\Support\Arr::map(Array, Object(Closure))
#3 /var/www/html/plugins/renatio/maillog/listeners/LogEmail.php(50): Illuminate\Support\Collection->map(Object(Closure))
#4 /var/www/html/plugins/renatio/maillog/listeners/LogEmail.php(26): Renatio\MailLog\Listeners\LogEmail->getAttachments(Object(Illuminate\Mail\Message))
#5 /var/www/html/vendor/laravel/framework/src/Illuminate/Events/Dispatcher.php(441): Renatio\MailLog\Listeners\LogEmail->handle(Object(October\Rain\Mail\Mailer), 'october.templat...', Object(Illuminate\Mail\Message))
#6 /var/www/html/vendor/laravel/framework/src/Illuminate/Events/Dispatcher.php(249): Illuminate\Events\Dispatcher->Illuminate\Events\{closure}('mailer.prepareS...', Array)
#7 /var/www/html/vendor/october/rain/src/Events/PriorityDispatcher.php(94): Illuminate\Events\Dispatcher->dispatch('mailer.prepareS...', Array, true)
#8 /var/www/html/vendor/october/rain/src/Events/PriorityDispatcher.php(61): October\Rain\Events\PriorityDispatcher->fireEvent('mailer.prepareS...', Array, true)
#9 /var/www/html/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(338): October\Rain\Events\PriorityDispatcher->fire('mailer.prepareS...', Array, true)
#10 /var/www/html/vendor/october/rain/src/Mail/Mailer.php(113): Illuminate\Support\Facades\Facade::__callStatic('fire', Array)
#11 /var/www/html/vendor/laravel/framework/src/Illuminate/Mail/MailManager.php(554): October\Rain\Mail\Mailer->send('october.templat...', Array, Object(Closure))
#12 /var/www/html/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(338): Illuminate\Mail\MailManager->__call('send', Array)
...
```

Additional information:
I use the [`attachData()`](https://docs.octobercms.com/3.x/extend/system/sending-mail.html#building-the-message) method to attach a calendar file.

Plugin Version: 1.0.4
October Build: 3.2.70
PHP Version: 8.0
Filesystem Driver: AWS S3

**Comment by triasrahman** [comment]
Hello @mplodowski , any update?

**Comment by mplodowski** [comment]
It is the issue with PHP 8.0 and composer pulling older version of symfony/mime package for Laravel.

It should be fixed in v1.0.5.
